CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 #### Improvements
 
 - Improved `to_pandas` to persist the original timezone offset for TIMESTAMP_TZ type.
+- Use SCOPED object for internal create temp tables.
```

Comment:
maybe explain what this means here because it is not in open doc.

Reply:
+1

Reply:
added more contents